### PR TITLE
fix row height

### PR DESCRIPTION
**Pull request by loucass003**

### CODE REVIEW

**Review by ImUrX — CHANGES_REQUESTED** [review]
fix the eslint problem, but seems fine

**Comment by loucass003** [comment]
No ? as long as the git lint config does not match the local files gui config. i wont follow it. And i wont fix it in this pr this is not the purpose of it

File: gui/src/components/tracker/TrackersTable.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
+import { useLocalization } from '@fluent/react';
 import classNames from 'classnames';
 import { IPv4 } from 'ip-num/IPNumber';
-import { MouseEventHandler, ReactNode, useState, useMemo } from 'react';
+import { MouseEventHandler, ReactNode, useMemo, useState } from 'react';
 import {
   TrackerDataT,
   TrackerIdT,
-  TrackerStatus as TrackerStatusEnum,
+  TrackerStatus as TrackerStatusEnum
```
Comment (ImUrX) [inline gui/src/components/tracker/TrackersTable.tsx:8]:
```suggestion
  TrackerStatus as TrackerStatusEnum,
```

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: gui/src/components/tracker/TrackersTable.tsx (hunk 1 above)
Comment (loucass003) [reply gui/src/components/tracker/TrackersTable.tsx:8]: no

**Review by ImUrX — APPROVED** [review]

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**